ccmlib/scylla_cluster.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, path, name, partitioner=None, install_dir=None,
         install_func = common.scylla_extract_install_dir_and_mode
 
         cassandra_version = kwargs.get('cassandra_version', version)
-        docker_image = kwargs.get('docker_image')
+        docker_image = kwargs.pop('docker_image', None)
```

Comment:
why are you using pop here ?

Reply:
I dont know there is `is_docker` method and I passed it tho all Node object (I revert this change)

Reply:
i still see it is `poping` it here

Reply:
Thanks, done :-)